packages/spark-core/src/plugins/credentials/authorization.js

```diff
@@ -37,7 +37,8 @@ const AuthorizationBase = SparkPlugin.extend({
         `expires`
       ],
       fn() {
-        return Boolean(this.access_token && this.expires && Date.now() > this.expires);
+        // if we don't have an access token, it can't actually be expired
+        return !this.access_token || Boolean(this.access_token && this.expires && Date.now() > this.expires);
```

Comment:
Without the new addition, it would be set as expired even if there was no access token (which may happen if there is a non-expired refresh_token and no access token). I'm not entirely sure this change is right longterm, but split token hasn't landed in 0.6, so the whole credentials plugin is likely to see a lot of adjustment in the next few weeks.